src/html/fetch-content.js

```diff
@@ -64,8 +65,17 @@ async function fetchContent(context, {
     }
 
     content.body = res.body;
+    setdefault(content, 'sources', []).push(downloader.computeGithubURI(owner, repo, branch, path));
 
-    setdefault(context.content, 'sources', []).push(downloader.computeGithubURI(owner, repo, branch, path));
+    // store extra source location if present
+    const sourceLocation = res.headers.get('x-source-location') || '';
+    const sourceHash = crypto.createHash('sha1').update(sourceLocation).digest('base64').substring(0, 16);
```

Comment:
Isn't there a helper in `helix-shared`?

Reply:
no. this is internal to pipeline. it's not used elsewhere.